guides/infra/bedrock-infra-rollback.md

```diff
@@ -0,0 +1,221 @@
+# Terraform Rollback Implementation using Bedrock CLI
+
+| Revision | Date         | Author         | Remarks       |
+| -------: | ------------ | -------------- | ------------- |
+|      0.1 | Apr-20, 2020 | Nathaniel Rose | Initial Draft |
+|      0.1 | May-11, 2020 | Nathaniel Rose | Updated Draft |
+
+## 1. Overview
+
+It is highly desireable to have a fully-automated Continuous Delivery pipeline
+in place to assist with testing and validation of source code changes made to
+the application services and supporting infrastructure. Without such a system,
```

Comment:
The first sentence is too long. 
suggestion: A Continuous Delivery pipeline automates testing and validation of source code changes made to the application services and supporting infrastructure.
Or some other rewording.

Reply:
Changed.